src/engine/passwordless/social_or_email_login_screen.jsx

```diff
@@ -45,12 +49,20 @@ const Component = ({ i18n, model }) => {
 
   const separator = social && email ? <PaneSeparator /> : null;
 
+  const captchaPane =
+  l.captcha(model) &&
+  l.captcha(model).get('required') &&
+  (isHRDDomain(model, databaseUsernameValue(model)) || !sso) ? (
```

Comment:
When I test this out in the browser, I get a runtime error on this line because `sso` is not defined. [In other places](https://github.com/auth0/lock/blob/7c9239e727b52bd5b3828b4bbed128c32ddab2dc/src/engine/classic/sign_up_screen.jsx#L29), we get this from `isSSOEnabled`.

However, the tests don't pick up this failure (they all pass) so looks like we're missing some coverage here. Could you take a look?